hutch_python/tests/test_happi.py

```diff
@@ -2,11 +2,12 @@
 import os.path
 import tempfile
 
+import pytest
 import simplejson
 
 from hutch_python.happi import get_happi_objs, get_lightpath
 
-from .conftest import beamlines, requires_lightpath
+from .conftest import beamlines, requires_lightpath, sources
```

Comment:
Not required to change but my opinion is:
* Generic names getting imported from some module are generally not so great
* Instead, importing the module and using `module.generic_name` is preferable

e.g., `conftest.sources`, `conftest.beamlines`